tasks/parallel.js

```diff
@@ -52,6 +65,19 @@ module.exports = function(grunt) {
         }
       });
     }
+    
+    if (options.config) {
+        // overtake some properties from the grunt config object (will be merged back again after this module loads)
+        var configObj={};
+        options.config.forEach(function ( configProp ) {
+            configObj[configProp] = grunt.config.get(configProp);
+        });
+        
+        flags.push('--parallelconfig=' + JSON.stringify(configObj));
```

Comment:
what if the stringified `configObj` contains whitespaces? will it still be parsed correctly?

Reply:
Yeah it should work, as there is some escaping mechanism for the args in grunt.spawn...

Reply:
Well, grunt itself doesn't do any extra escaping (https://github.com/gruntjs/grunt-legacy-util/blob/master/index.js#L185)
Perhaps Node.js does it, though.

Could you please test that to make sure it's supported nicely?

Reply:
Tested and works :-)
```javascript
space_test: "foo bar",
        space_test_1: { test: "" },
        
        parallel: {
            space_test: {
                options: {
                    grunt: true,
                    options: ['space_test']
                },
                tasks: ['space_test_1:test']
            }
}

grunt.registerMultiTask('space_test_1', 'space test', function() {
        grunt.log.write("value of config " + grunt.config.get('space_test'));
});
```

**Output:**
```
Jans-MBP:Frontend jan$ grunt parallel:space_test
Running "parallel:space_test" (parallel) task

    Running "space_test_1:test" (space_test_1) task
    value of config foo bar
    Done, without errors.
    
Done, without errors.
```

Reply:
Thanks!